### DISCUSSION TITLE
Remove Config

### DISCUSSION THREAD

**Discussion by shingsam** [homeassistant-apps/app-cloudflared#268] [Q&A] [answered]
Hello, can u add menu to delete old file config or edit config?
Cause when i change domain name will crash.

Thanks,

**Comment by brenner-tobias** [comment, marked as answer]
We used to have this option to delete all the Cloudflared files. The current process is that the config is created in a temporary folder and re-created everytime you start the add-on, so there should not be any issues. If you do have problems, I suggest to uninstall the add-on and re-installing it.

If you are talking about the HA Add-On Config, you can change that in the Add-On configuration by navigating to the tab "Configuration" of the Add-On, clicking on the three dots on the right top corner and choosing "Edit in YAML".
